packages/vite-plugin-checker/src/types.ts

```diff
@@ -19,6 +20,11 @@ interface TsConfigOptions {
    * root path of cwd
    */
   buildMode: boolean
+
+  /**
+   * Additional Compiler options to merge with tsconfig
+   */
+  extraTSOptions: ts.CompilerOptions
```

Comment:
**question:** Are docs changed needed here @EliLichtblau?

Reply:
I can change the docs if wanted - I thought the comment would be clear in intellisense - I think the mistake here is I didn't make "extraTSOptions" optional which was an oversight

Reply:
Should be added to the docs [here](https://github.com/fi3ework/vite-plugin-checker/blob/main/docs/checkers/typescript.md) I think.

Reply:
thanks! will do now

Reply:
Added in https://github.com/fi3ework/vite-plugin-checker/pull/311/commits/ee392c773fb769da6fada676b204a8157867d986